mirror_lkl_layer.md

```diff
@@ -0,0 +1,55 @@
+## Mirror Last Known Locations layer (LKL) into a standard feature service
+
+This script will copy data from a Last Known Locations (LKL) layer in a Location Tracking layer and mirror it into a standard feature service. 
+
+As of April 2020, the Last Known Locations layer does not support dynamic joins, so the mirroring of data using this script may be useful if the user is trying to relate additional data to their last known locations data. Standard feature services support join features.
+
+This script is designed to be run as a scheduled task, at whatever frequency you want to get updated LKL data from your Location Tracking layer. For example, the admin may set up this script to run every 10 minutes in order to get the latest data from the field.
+
+For information on how to set up scheduled tasks, please see: https://www.esri.com/arcgis-blog/products/arcgis-pro/analytics/schedule-a-python-script-or-model-to-run-at-a-prescribed-time-2019-update/
+
+and https://www.esri.com/arcgis-blog/products/product/analytics/scheduling-a-python-script-or-model-to-run-at-a-prescribed-time/
+
+This script requires the use of a point layer that the data can be mirrored into. This layer would ideally share all the same fields as a standard LKL layer. We have a publicly shared layer with the requisite fields ready for you to import into your organization using the `clone_items` functionality in the Python API.
+
+To import this layer into your organization (in AGOL), do:
+
+```python
+import arcgis
+gis = arcgis.gis.GIS("https://arcgis.com","myusername","mypassword")
+mirror_layer_item = gis.content.get("id")
+gis.content.clone_items([mirror_layer_item])
+```
+
+This will create a standard feature service in your organization whose item id you can pass into this script. The layer you are cloning here is hosted in ArcGIS Online. To use this script in Enterprise, you also must prepare a layer to have mirrored data. Either
```

Comment:
Could you clone the location tracking service lkl layer directly?
```python
items = gis.content.search('typekeywords: "Location Tracking Service" NOT typekeywords: "Location Tracking View")
if items:
    gis.content.clone_items([item[0])
```

I guess this would clone both layers, but could add an updateDefinition call to remove the tracks layer. 

I think you should investigate this approach since it will mean we don't need to maintain the sample layer.